index.html

```diff
@@ -4,31 +4,38 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We"
+      crossorigin="anonymous"
+    />
+
     <link rel="stylesheet" href="/assets/styles/styles.css" />
     <script src="/assets/js/logic.js" defer></script>
     <title>Awesome Books</title>
   </head>
 
   <body>
-    <header>
-      <h1>Awesome books</h1>
-    </header>
-    <br />
-    <main>
-      <section>
-        <h3>Book List</h3>
-        <ul id="main-list"></ul>
-      </section>
-      <hr />
-      <br />
-      <section>
-        <h3>Add Books Here</h3>
-        <form action="submit" id="main-form">
-          <input type="text" name="Title" placeholder="Title" id="title" />
-          <input type="text" name="Author" placeholder="Author" id="author" />
-          <input type="submit" value="submit" />
-        </form>
-      </section>
-    </main>
+    <div class="container">
+      <main class="container-fluid">
+        <div id="listDiv" class="mt-3">
+          <h1 class="text-center my-4">All Awesome Books</h1>
+          <ul
+            id="main-list"
+            class="book-list m-auto border border-dark ps-0 my-4"
+          ></ul>
+        </div>
+
+        <div id="formDiv" class="my-4">
+          <form action="submit" id="main-form" class="m-auto add-form">
+            <h2 class="text-center my-4">Add a New Book</h2>
+            <input type="text" name="Title" placeholder="Title" id="title" />
+            <input type="text" name="Author" placeholder="Author" id="author" />
+            <input type="submit" value="submit" class="btn btn-secondary" />
```

Comment:
- [ ] There should be some margin between the input field and submit button as shown in the wireframe. Kindly try to fix it.
Your
![image](https://user-images.githubusercontent.com/53019916/129468315-7358c9b7-3fbe-470d-b690-1e7b2ae4b06d.png)

Original  
![image](https://user-images.githubusercontent.com/53019916/129468327-ddaf5573-0e75-4568-939e-bf6ef4459101.png)

Reply:
This issue is well resolved! ✔️